portfolio/src/main/webapp/index.html

```diff
@@ -58,27 +58,46 @@ <h1>Kwame Ampadu-Boateng's Portfolio</h1>
         Click the button below to get a random message. The message could either
         be about me or a movie/book quote that I enjoy.
       </p>
-      <button onclick="addRandomGreeting()">Explore</button>
+      <button onclick="addRandomGreeting()" class="regButton">Explore</button>
       <div id="greeting-container"></div>
 
       <br />
 
-      <button onclick="getData()">Get Hello</button>
+      <button onclick="getData()" class="regButton">Get Hello</button>
       <div id="data-container"></div>
 
       <br />
 
       <form action="/jsonData" method="POST">
         <label for="userComment">Add a comment:</label><br />
         <input type="text" id="userComment" name="userComment" /><br />
-        <br /><br />
         <input type="submit" />
       </form>
 
       <br />
 
-      <button onclick="getJson()">See Comments.</button>
+      <form action="/jsonData">
+        <p>How many comments do you want to see?</p>
+        <input
+          type="number"
+          name="maxCount"
+          min="0"
+          max="50"
+          value="10"
+          id="maxForm"
+          onchange="getJson(this.value)"
```

Comment:
Does the `onchange` do anything here? Doesn't look like `getJson` takes a parameter.

Reply:
I just tested it out. Onchange does do what it is suppose to do; however,  "this.value" does not impact the functionality so I will remove it.